src/ui/layout/index.md

```diff
@@ -188,6 +188,71 @@ class MyApp extends StatelessWidget {
   or you can build your own set of custom widgets.
 {{site.alert.end}}
 
+#### Cupertino apps
+
+To create a `Cupertino` app, use a [`CupertinoPageScaffold`][] widget.
+
+Unlike `Material`, it doesn't provide a default banner or background color.
+You need to set these yourself.
+
+* To set default colors, you add a [`CupertinoThemeData`][].
+* To add a [`CupertinoNavigationBar`][] as a `navigationBar` property value.
+  You can modify per widget colors using [`CupertinoColors`][].
+
+* To layout the body of your app, set a `child` property with the
+  desired widget as its value, like `Center` or `Column`.
+
+To learn what other UI components you can add, check out the
+[Cupertino library][].
+
+<?code-excerpt path-base="layout/base"?>
```

Comment:
I don't think you need this since the path base was already set to "layout/base" above.

```suggestion
```